### DISCUSSION TITLE
DNS cache configuration

### DISCUSSION THREAD

**Discussion by lbtboy** [AdguardTeam/AdGuardHome#3908] [Q&A] [answered]
**AdGuardHome Version: v0.106.3**

Приветствую

![AdGuard Home - 01 12 2021_16_42_03](https://user-images.githubusercontent.com/57795912/144229134-8f3fa383-b40e-4519-b950-e4cf3d7612e3.jpg)
 

**Подскажите какой оптимальный размер параметра `Настройка кэша DNS`, куда кэшируются данные и где хранятся? Как правильно настроить размер, если есть какие либо мануалы**

**Comment by ainar-g** [comment, marked as answer]
Добрый день.

Размер зависит от типа нагрузки.  Если разных доменов в сети запрашивается много, лучше поставить больше размера по умолчанию.

Данные кэшируются в оперативной памяти, так что размер памяти на сервере стоит учитывать при выборе размера кэша.

**Reply by lbtboy** [reply]
> Добрый день.
> 
> Размер зависит от типа нагрузки. Если разных доменов в сети запрашивается много, лучше поставить больше размера по умолчанию.
> 
> Данные кэшируются в оперативной памяти, так что размер памяти на сервере стоит учитывать при выборе размера кэша.

Если размер 1gb на сервере и используется всего 374мб могу я установить например параметр 16777216 и насколько это нагрузит систему в том числе и память?

**Reply by ainar-g** [reply]
Да, можете.  В теории должно занять приблизительно столько же памяти и снизить нагрузку на i/o.

**Reply by lbtboy** [reply]
> Да, можете. В теории должно занять приблизительно столько же памяти и снизить нагрузку на i/o

Спасибо за исчерпывающий ответ!

.
